CMakeLists.txt

```diff
@@ -57,32 +59,46 @@ find_package(nanobind CONFIG REQUIRED)
 #  src/nanobind_example_ext.cpp
 #)
 
-nanobind_add_module(pcl_common_ext src/python_common.cpp)
-target_link_libraries(pcl_common_ext PRIVATE pcl_common)
+# Setup for linux, see information on
```

Comment:
I got errors with the original linking in the ubuntu-docker container. Following this setup resolved that.

Reply:
What was the linking error exactly? The approach with setting INSTALL_RPATH to /usr/local/lib only works on Linux (maybe macOS?) and probably only if the PCL libraries are installed there?